blocks/footer/footer.css

```diff
@@ -1,14 +1,267 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+footer .footer-wrapper {
+  background: #fcfcfc;
 }
 
-footer .footer {
-  max-width: 1200px;
+footer .section.footer-top-note {
+  padding-top: 0;
+  padding-bottom: 1.25rem;
+}
+
+.footer-links .section-container{
+  padding-left: 3.438rem;
+  padding-right: 3.438rem;
+  border-bottom: 0.0625rem solid #979797;
+}
+
+.footer-disclaimer .section-container{
+  padding-left: 0.9375rem;
+  padding-right: 0.9375rem;
+  padding-bottom: 1.875rem;
+}
+
+.section.footer-top-note .section-container{
+  padding-left: 1rem;
+}
+
+.footer-top-note p {
+  color: #0e1f35;
+  font-size: 0.8125rem;
+}
+
+.footer-top .toll-free-number {
+  color: #004c84;
+  font-size: 2rem;
+  font-weight: 700; 
+  line-height: 3rem;
+  padding-left: 1rem; 
+}
+
+.section .footer-top-wrapper {
+  position: relative;
+  z-index: 99;
```

Comment:
woudn't the z-index 1 work here, we can keep it to minimal required value

Reply:
agreed, Done